src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -0,0 +1,692 @@
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTable;
+import javax.swing.SwingConstants;
+import javax.swing.border.EmptyBorder;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.DefaultTableModel;
+
+import org.isf.generaldata.MessageBundle;
+import org.isf.medicalinventory.gui.InventoryEdit.InventoryListener;
+import org.isf.medicalinventory.manager.MedicalInventoryManager;
+import org.isf.medicalinventory.model.InventoryStatus;
+import org.isf.medicalinventory.model.InventoryType;
+import org.isf.medicalinventory.model.MedicalInventory;
+import org.isf.menu.manager.Context;
+import org.isf.utils.exception.OHServiceException;
+import org.isf.utils.exception.gui.OHServiceExceptionUtil;
+import org.isf.utils.jobjects.GoodDateChooser;
+import org.isf.utils.jobjects.MessageDialog;
+import org.isf.utils.jobjects.ModalJFrame;
+import org.isf.utils.time.TimeTools;
+import org.springframework.data.domain.Page;
+
+public class InventoryBrowser extends ModalJFrame implements InventoryListener {
+
+	private static final long serialVersionUID = 1L;
+	private GoodDateChooser jCalendarTo;
+	private GoodDateChooser jCalendarFrom;
+	private LocalDateTime dateFrom = TimeTools.getNow();
+	private LocalDateTime dateTo = TimeTools.getNow();
+	private JLabel jLabelTo;
+	private JLabel jLabelFrom;
+	private JPanel panelHeader;
+	private JPanel panelFooter;
+	private JPanel panelContent;
+	private JButton jButtonClose;
+	private JButton jButtonNew;
+	private JButton jButtonEdit;
+	private JButton jButtonDelete;
+	private JButton jButtonView;
+	private JScrollPane scrollPaneInventory;
+	private JTable jTableInventory;
+	private String[] columsNames = {
+			MessageBundle.getMessage("angal.common.reference.label").toUpperCase(),
+			MessageBundle.getMessage("angal.common.date.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.common.status.txt").toUpperCase(),
+			MessageBundle.getMessage("angal.common.user.col").toUpperCase() };
+	private int[] columwidth = { 150, 150, 150, 200 };
+	private boolean[] columnCentered = { false, true, true, true };
+	private JComboBox<String> statusComboBox;
+	private JLabel statusLabel;
+	private JButton next;
+	private JButton previous;
+	private JComboBox<Integer> pagesComboBox = new JComboBox<>();
+	private JLabel ofPagesLabel = new JLabel(MessageBundle.formatMessage("angal.common.pages.fmt.txt"), 1);
+	private static int PAGE_SIZE = 24;
+	private int startIndex = 0;
+	private int totalRows;
+	private MedicalInventoryManager medicalInventoryManager = Context.getApplicationContext().getBean(MedicalInventoryManager.class);
+	private List<MedicalInventory> inventoryList;
+
+	public InventoryBrowser() {
+		initComponents();
+	}
+
+	private void initComponents() {
+		setDefaultCloseOperation(JFrame.DISPOSE_ON_CLOSE);
```

Comment:
```suggestion
		setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
```
Add import for `javax.swing.WindowConstants` too.